fake/src/faker/impls/uuid.rs

```diff
@@ -0,0 +1,126 @@
+use uuid::{Builder, Uuid, Variant, Version};
+
+use crate::faker::uuid::raw::*;
+use crate::locales::Data;
+use crate::{Dummy, Faker};
+
+impl<L: Data> Dummy<UuidV1<L>> for Uuid {
+    fn dummy_with_rng<R: rand::Rng + ?Sized>(config: &UuidV1<L>, rng: &mut R) -> Self {
+        let bytes: [u8; 8] = rng.gen();
+        let (ticks, _) = config.1.to_rfc4122();
+        let time_low = (ticks & 0xFFFF_FFFF) as u32;
+        let time_mid = ((ticks >> 32) & 0xFFFF) as u16;
+        let time_high_and_version = (((ticks >> 48) & 0x0FFF) as u16) | (1 << 12);
+        Uuid::from_fields(time_low, time_mid, time_high_and_version, &bytes)
+            .expect("generate uuid::Uuid::from_fields")
+    }
+
+    fn dummy(UuidV1(_, timestamp, node_id): &UuidV1<L>) -> Self {
+        Self::new_v1(*timestamp, node_id).expect("generate uuid::Uuid::new_v1")
```

Comment:
seems this will always produce same Uuid?

Reply:
yes indeed I'll check this out